packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -1,10 +1,9 @@
-import React, { useState, ChangeEvent } from "react";
+import { Select } from "@fremtind/jkl-dropdown-react";
+import { TextField } from "@fremtind/jkl-text-input-react";
+import { SupportLabel } from "@fremtind/jkl-typography-react";
 // @ts-ignore
 import CoreDatepicker from "@nrk/core-datepicker/jsx";
-// @ts-ignore
-import CoreToggle from "@nrk/core-toggle/jsx";
-import { TextField } from "@fremtind/jkl-text-input-react";
-import { Select } from "@fremtind/jkl-dropdown-react";
+import React, { ChangeEvent, MutableRefObject, useEffect, useRef, useState } from "react";
```

Comment:
Can we move this to the top? My OCD is kicking in when external deps come after internal ones 😅

Reply:
Move away :)